traitcuration/traits/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+
+class Trait(models.Model):
+    name = models.CharField(max_length=200)
+    current_mapping = models.ForeignKey(
+        'Mapping', on_delete=models.SET_NULL, null=True, blank=True)
+    status = models.CharField(max_length=50)
```

Comment:
Just to clarify: the status is a `CharField` only because this is a prototype, right? In the future it would have to become a `ComputedField`

Reply:
Also I just want to make sure that either there is already a ticket for implementing this part in the backend (in this case it should be linked under this comment), or not yet (in which case I'll create one)

Reply:
Yes that is so I can manually add a status via the Django Admin interface, but I have been thinking about that. There is no built-in `ComputedField` option in the Django ORM. There are two solutions to that. 
- Keep it as a CharField and add a check whenever a review is issued or whenever a mapping is changed to automatically compute the field's value
- Use this package https://pypi.org/project/django-computedfields/

Reply:
I see. This looks like it can be a whole separate issue, I've created one: https://github.com/EBIvariation/trait-curation/issues/41. Let's use it to evaluate all those possible approaches.